aopy/visualization.py

```diff
@@ -892,3 +892,18 @@ def plot_boxplots(data, plt_xaxis, trendline=True, facecolor=[0.5, 0.5, 0.5], li
             boxprops=dict(facecolor=facecolor, color=linecolor), capprops=dict(color=linecolor),
             whiskerprops=dict(color=linecolor), flierprops=dict(color=facecolor, markeredgecolor=facecolor),
             medianprops=dict(color=linecolor))
+
+def get_plotcolors(nitems, colormap="viridis"):
```

Comment:
maybe set_plotcolors()? i was imagining a function that returns a list of colors, for example to be used in color_trajectories(), above. but we can change things around so that function just grabs colors from the current pallette.